articles/install-guide/csinstall.md

```diff
@@ -135,7 +136,86 @@ Of course, you can also build and run Q# programs from the command line by simpl
 
         You should see the following output: `Hello quantum world!`
 
+## Q# with other .NET languages <a name="dotnet"></a>
+
+Q# is compiled to C# for the purposes of local simulation. This makes them very easy to combine. However, you can also use other .NET languages like F# or VB\.NET, although the process is less straightforward.
+
+### Develop with Q# + F# creating a Q# library
+
+You can't mix Q# and F# files like we do with Q# and C#. To combine F# with Q# you need to create a Q# library which will yield a .csproj project with only Q# files in it. Then you need to reference it from a purely F# application.
+
+1. Pre-requisites
+
+    - [.NET Core SDK 3.1 or later](https://www.microsoft.com/net/download)
+
+1. Create a Q# library `QuantumCode` and write your quantum code in it.
+
+1. Create an F# application (in this case a console app targeting .NET Core) `FsharpDriver`.
```

Comment:
It may be helpful here to point out that the same `dotnet new` command works for F# projects as well, with a different option to `-lang`.

Reply:
Sorry, I thought that comment was published but has been "pending" almost 20 days :/

Reply:
I've definitely done that before... easy enough to do!